src/components/MainLayout/MainHeader.tsx

```diff
@@ -206,17 +258,65 @@ export const MainHeader = () => {
       {suspendedButtons}
     </div>
   ) : (
-    <div className={classes.root}>
+    <Box className={classes.root}>
```

Comment:
What is the improvement of having Box here?

Reply:
Because we're using MUI allows to us to use that useful props so it's much shorter than create class when I will use different styles for each boxes, instead of create a class that would be adds padding, flex props etc and maybe will be unique for that part its better to use it, maybe in this particular Box I can take away some of the css props to get a better implementation as in the other implementations.

Reply:
As none of those props are used here - it's more performant and simpler to use HTML.